spec/features/admins/admin_sees_merchant_dashboard_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+RSpec.describe 'admin views merchant dashboard' do
+  describe 'as an admin when Im on the merchant index page and I click on a merchant name' do
+    before :each do
+      @merchant = create(:merchant)
+      admin = create(:admin)
+
+      allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(admin)
+    end
+
+    it 'takes me to the route admin/merchants/:id' do
+      visit merchants_path
+
+      click_on "#{@merchant.name}"
+
+      expect(current_path).to eq(admin_merchant_path(@merchant))
+    end
+
+    # it 'shows the same info the merchant sees' do
```

Comment:
Why is this test commented out? Is it for a feature that hasn't been written yet?

Reply:
Yes this whole file is the first half of a user story I started on a different branch, but I needed the merchant dashboard to complete. I'll finish it as soon as I can merge the merchant dashboard.